client/js/middleware/qbank.js

```diff
@@ -220,10 +221,18 @@ const qbank = {
       if (!_.isEmpty(answers)) {
         newItem.answers = answers;
       }
-      if (!_.isEmpty(choices) || item.questionString) {
+      if (!_.isEmpty(choices)
+        || updatedItem.questionString
+        || updatedItem.question.maintainOrder
+        || updatedItem.question.maintainOrder === false) {
         newItem.question = {};
-        if (item.questionString) { newItem.question.questionString = item.questionString; }
+        if (updatedItem.questionString) {
+          newItem.question.questionString = updatedItem.questionString;
+        }
         if (!_.isEmpty(choices)) { newItem.question.choices = choices; }
+        if (updatedItem.question.maintainOrder || updatedItem.question.maintainOrder === false) {
```

Comment:
are you checking if `maintainOrder` is either not null or false? Could this be replaced with a single check to see if it is not null?